backend/src/ai_ticket_platform/services/infra/storage/azure.py

```
@@ -113,3 +113,53 @@ def put_presigned_url(
+			logger.info(f"Successfully uploaded blob: {self.container_name}/{blob_name}")
+			return blob_name
+		except Exception as e:
+			logger.error(f"Failed to upload blob {blob_name}: {e}", exc_info=True)
+			raise
+
+	def download_blob(self, blob_name: str, decode: bool = True) -> Union[str, bytes]:
+		"""
+		Download content from Azure Blob Storage.
+
+		Args:
+		    blob_name: Name/path of the blob
```

Ruff (0.14.7)

Indentation contains mixed spaces and tabs

(E101)